go.mod

```
@@ -16,6 +18,7 @@ require (
 	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v7 v7.1.2
 	github.com/cosmos/ibc-go/v7 v7.3.2
 	github.com/cosmos/interchain-security/v3 v3.3.0
+	github.com/crypto-org-chain/cronos/store v0.0.0-00010101000000-000000000000
```

```suggestion
	github.com/crypto-org-chain/cronos/store v0.0.5-0.20240301012710-e9bb006c8d14
```

I think you can pin the version here, and remove the replacement.

---

Will try again.  go mod tidy kept on putting zeros.

---

will be able to fix after removing the replace

---

fixed